packages/starlight/components/Select.astro

```diff
@@ -17,7 +17,7 @@ interface Props {
 <label style={`--sl-select-width: ${Astro.props.width}`}>
 	<span class="sr-only">{Astro.props.label}</span>
 	<Icon name={Astro.props.icon} class="icon label-icon" />
-	<select value={Astro.props.value}>
+	<select value={Astro.props.value} autocomplete="off">
```

Comment:
The intention is still to add the `autocomplete` attribute here right? Just for the browsers/conditions where it does work?

May be worth a comment inline to remind ourselves why we added it.